docs/tsc/meetings/2020-05-11.md

```diff
@@ -0,0 +1,69 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+May 11, 2020
+
+Host: Michael Dolan
+
+Rotating Secretary: Doug Walker
+
+Attendees:
+  * [x] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [x] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [ ] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [x] Michael Dolan (_TSC Chair_) - Epic Games
+  * [x] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [x] Carol Payne (_TSC_) - Netflix
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [x] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [x] Kevin Wheatley (_TSC_) - Framestore
+  * Joseph Goldstone - ARRI
+  * Bernard Lefebvre - Autodesk
+
+Apologies:
+  Sean Cooper
+
+# **OCIO TSC Meeting Notes**
+
+* Videos for OCIO
+    - Michael: Emily Olin (ASWF PR director) is hoping to help us develop some videos explaining 
+      and demonstrating OCIO, and is looking for studios who can contribute recent images, videos, 
+      breakdowns etc. that could be used within that. If any of you have some content you could 
+      clear, let me know and I can put Emily in contact with you.
+    - Michael: I was able to get a few things cleared when I was at Sony, but want some new content.
```

Comment:
```suggestion
    - Michael: I was able to get a few things cleared when I was at Sony, but it would be great to 
      get some new content in addition to that.
```